win32/win32.c

```diff
@@ -2664,15 +2664,23 @@ set_pioinfo_extra(void)
 # else /* x86 */
     /* pop ebp */
 #  define FUNCTION_BEFORE_RET_MARK "\x5d"
+#  ifdef _UCRT
+     /* leave */
+#   define FUNCTION_BEFORE_RET_MARK_2 "\xc9"
+#  endif
 #  define FUNCTION_SKIP_BYTES 0
     /* mov eax,dword ptr [eax*4+100EB430h] */
 #  define PIOINFO_MARK "\x8B\x04\x85"
 # endif
     if (p) {
-        for (pend += 10; pend < p + 300; pend++) {
+        for (pend += 10; pend < p + 500; pend++) {
             // find end of function
-            if (memcmp(pend, FUNCTION_BEFORE_RET_MARK, sizeof(FUNCTION_BEFORE_RET_MARK) - 1) == 0 &&
-                (*(pend + (sizeof(FUNCTION_BEFORE_RET_MARK) - 1) + FUNCTION_SKIP_BYTES) & FUNCTION_RET) == FUNCTION_RET) {
+            if ((memcmp(pend, FUNCTION_BEFORE_RET_MARK, sizeof(FUNCTION_BEFORE_RET_MARK) - 1) == 0
+# ifdef _UCRT
```

Comment:
This breaks `_WIN64` build.

```suggestion
# ifdef FUNCTION_BEFORE_RET_MARK_2
```

Reply:
i think it fails because of missing bracket, introduced by the new patch.. i will try fixing it soonish